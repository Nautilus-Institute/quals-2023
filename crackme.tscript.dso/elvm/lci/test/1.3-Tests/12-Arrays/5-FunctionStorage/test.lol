HAI 1.3
	HOW IZ I fun1
		VISIBLE "a"
	IF U SAY SO

	HOW IZ I fun2 YR a
		VISIBLE "b" AN a
	IF U SAY SO

	HOW IZ I fun3 YR a AN YR b AN YR c
		VISIBLE "c" AN a AN b AN c
	IF U SAY SO

	I HAS A foo ITZ A BUKKIT
	foo HAS A var1 ITZ fun1
	foo HAS A var2 ITZ fun2
	foo HAS A var3 ITZ fun3

	I IZ foo'Z var1 MKAY
	I IZ foo'Z var2 YR 1 MKAY
	I IZ foo'Z var3 YR 1 AN YR 2.345 AN YR "xyz" MKAY
KTHXBYE
