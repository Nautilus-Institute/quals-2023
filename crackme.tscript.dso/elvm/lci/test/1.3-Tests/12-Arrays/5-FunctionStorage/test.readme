This test is designed to check that functions may be stored in array slots.
