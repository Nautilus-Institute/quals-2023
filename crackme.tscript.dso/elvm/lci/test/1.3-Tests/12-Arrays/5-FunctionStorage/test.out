a
b1
c12.34xyz
